src/site/content/en/progressive-web-apps/manifest-updates/index.md

```diff
@@ -104,18 +100,12 @@ used.
 
 * `background_color`
 * `display`
-* `icons`
-* `name`
 * `orientation`
 * `scope`
 * `shortcuts`
-* `short_name`
 * `start_url`
```

Comment:
`start_url` is mentioned above as being excluded from update so probably should be removed here, too.

Reply:
it is excluded on desktop, but still supported on mobile, some changes are coming but will update the article when those happen.